docs/operators/fixedWidthParserOperator.md

```diff
@@ -0,0 +1,170 @@
+Fixed Width Parser Operator
+=============
+
+## Operator Objective
+This operator is designed to parse fixed width records and construct a map or concrete java class also known as ["POJO"](https://en.wikipedia.org/wiki/Plain_Old_Java_Object) out of it. User need to provide the schema to describe the fixed width data. Based on schema definition the operator will parse the incoming record to object map and POJO.
+Valid records will be emitted as POJO / map while invalid ones are emitted on error port with error message.
+
+**Note**: field names in schema must match field names of POJO and in the same order as it appears in the incoming data.
+
+FixedWidthParser is **idempotent**, **fault-tolerant** and **statically/dynamically partitionable**.
+
+## Class Diagram
+![](images/fixedWidthParser/fixedWidthParser.png)
+
+## Operator Information
+1. Operator location:**_malhar-contrib_**
+2. Available since:**_3.8.0_**
+3. Operator state:**_Evolving_**
+4. Java Package:[com.datatorrent.contrib.parser.FixedWidthParser](https://github.com/apache/apex-malhar/blob/master/contrib/src/main/java/com/datatorrent/contrib/parser/FixedWidthParser.java)
+
+
+## <a name="props"></a>Properties of FixedWidthParser
+User need to set the schema which describes fixed width data.
```

Comment:
Providing the actual XML fragment will be more useful than just the JSON, e.g.

The user needs to set the `jsonSchema` property to an appropriate JSON string that describes the fixed width data. An example of how to set it in the `properties.xml` file is shown below:
```xml
<property>
    <name>jsonSchema</name>
    <value> .... </value>
</property>
```